tools/certa/src/CertaConfig.ts

```diff
@@ -137,12 +137,8 @@ export namespace CertaConfig {
   function resolvePaths(baseDir: string, opts: PartialCertaConfig): void {
     for (const propPath of _filePathOpts) {
       const relativeFilePath = lodash.get(opts, propPath);
-      if (relativeFilePath) {
```

Comment:
@wgoehrig lodash released a new version 8 hours ago with [this change](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62968).
Compiler complained that:
```
src/CertaConfig.ts(142,55): error TS2339: Property 'map' does not exist on type 'never'.
src/CertaConfig.ts(142,60): error TS7006: Parameter 'p' implicitly has an 'any' type.
```

Please review.

Reply:
With this change, core-full-stack-tests throws ERR_INVALID_ARG_TYPE. There's a mismatch between type definitions and run-time types. I will revert.